static/src/stylesheets/module/site-messages/_first-pv-consent.scss

```diff
@@ -194,3 +194,19 @@ $btn-height: ($gs-baseline*3.5);
         }
     }
 }
+
+// commercialIabBottomConsentBanner A/B test styles
+
+.cmp-options-button {
+    height: 42px;
+    border: 0;
+    border-radius: 1000px;
+    padding: 0 20px;
+    margin-left: 8px;
+    background: #767676;
```

Comment:
Minor: we have lots of sass vars that can be used, this one would be: `background:  $brightness-46`

https://github.com/guardian/frontend/blob/master/static/src/stylesheets/_vars.scss

Reply:
👍